src/Plugin/EventSubscriberInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * @copyright 2017 Hostnet B.V.
+ */
+declare(strict_types=1);
+
+namespace Hostnet\Component\Resolver\Plugin;
+
+use Hostnet\Component\Resolver\Bundler\Pipeline\ContentPipelineInterface;
+use Hostnet\Component\Resolver\Import\ImportFinderInterface;
+
+/**
+ * Plugins should implement this interface if they require events attached to the pipeline.
```

Comment:
We should probably denote an example of what you can do with these events here

It also would help devs a lot to link to the related Symfony docs!